egs/tedlium/s5_r2/local/chain/tuning/run_tdnn_lstm_1q.sh

```diff
@@ -0,0 +1,348 @@
+#!/bin/bash
+
+# 1q is as 1p, but add the "dropout-exclusive" option which means that
```

Comment:
in the next week or two, please work on getting a 'clean' pull request, something that we can check in.
The aim is to demonstrate just our optimal recipe, and to get rid of any tuning scripts that are not things
we plan to use in future-- for example, this script, which uses the 'dropout-exclusive' option.

Check again what is the currently recommended script in each case, and make sure that the script you are checking in is modified from that, and uses our recommended dropout setting.  Let's aim to check in
good-performing systems on AMI, Tedlium and Swbd, based on fast TDNN+LSTMs (and probably also
BLSTMs).  In each case there should just be one new script for each type of setup, e.g. just one new script
for Switchboard TDNN+LSTMs.